src/olympia/blocklist/admin.py

```diff
@@ -534,6 +544,12 @@ def get_fieldsets(self, request, obj):
 
         return (details, history, edit)
 
+    def has_change_permission(self, request, obj=None):
+        if obj and BlockSubmission.get_submission_from_guid(obj.guid):
```

Comment:
Also, I asked this elsewhere, but I assume that `get_submission_from_guid` returns `None` if none are found and not an empty list?

Reply:
it returns a queryset which is falsey when there are no results.  (an empty list is also falsey in python)